pages/design-tokens/typesetting/font-family.md

```diff
@@ -156,10 +156,10 @@ Role-based tokens set the font family value based on the _role_ the face plays i
 </div>
 
 {:.bg-gold-20v.padding-2.radius-md}
-**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. This documentation is coming soon. See the inline documentation in `_uswds-theme-typography` for more details.
+**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. Create a new project-specific theme settings file. If your project already has a single project-specific theme settings file, you’re all set. If not, create a new file in your project theme file called `_uswds-theme.scss`.
```

Comment:
This could use a top margin.

![image](https://user-images.githubusercontent.com/3385219/200365311-fd155a82-fb00-4a1e-9afd-271d9a4455b9.png)